src/main/java/edu/umich/lib/dor/replicaexperiment/service/DepositFactory.java

```diff
@@ -0,0 +1,48 @@
+package edu.umich.lib.dor.replicaexperiment.service;
+
+import java.nio.file.Path;
+
+import edu.umich.lib.dor.replicaexperiment.domain.User;
+
+public class DepositFactory {
+    InfoPackageService infoPackageService;
+    RepositoryService repositoryService;
+    ReplicaService replicaService;
+    RepositoryClientRegistry repositoryClientRegistry;
+    Path depositPath;
+
+    public DepositFactory(
+        InfoPackageService infoPackageService,
+        RepositoryService repositoryService,
+        ReplicaService replicaService,
+        RepositoryClientRegistry repositoryClientRegistry,
+        Path depositPath
+    ) {
+        this.infoPackageService = infoPackageService;
+        this.repositoryService = repositoryService;
+        this.replicaService = replicaService;
+        this.repositoryClientRegistry = repositoryClientRegistry;
+        this.depositPath = depositPath;
+    }
```

Comment:
I found the most straightforward approach for combining injected dependencies and regular parameters was to use regular constructors and handle the wiring in `AppConfig`. There may be a better way though...